app/components/Application/ApplicationOverrideJustification.tsx

```diff
@@ -77,7 +79,9 @@ export const ApplicationOverrideJustificationComponent: React.FunctionComponent<
     setAccordionOpen(true);
   };
 
-  if (overrideActive) {
+  if (overrideJustification && !hasErrors) handleOverrideDelete();
```

Comment:
this should be in a `useEffect` hook. If an exception is thrown during the component's render, then the ErrorBoundary will be rendered, preventing the user from seeing the summary page.